components/Article/Page.js

```diff
@@ -411,7 +411,8 @@ class ArticlePage extends Component {
     return (
       <Frame
         raw
-        meta={meta}
+        // Meta tags for a focus comment are rendered in Discussion/Commments.js
+        meta={router.query.focus ? undefined : meta}
```

Comment:
check that `meta.discussionId` is present too, otherwise one could disable meta data on a non discussion by setting focus param that has no other effect